adapters/adview/adview.go

```diff
@@ -117,12 +136,14 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.R
 	}
 
 	bidResponse := adapters.NewBidderResponseWithBidsCapacity(len(request.Imp))
-	bidResponse.Currency = "USD" //we just support USD for resp
+	//we just support USD for resp
+	bidResponse.Currency = "USD"
 
 	var errors []error
 	for _, seatBid := range response.SeatBid {
 		for i, bid := range seatBid.Bid {
-			bidType, err := getMediaTypeForImp(bid.ImpID, request.Imp)
+			bidType, err := getMediaTypeForBid(bid, request.Imp)
+			//bidType, err := getMediaTypeForImp(bid.ImpID, request.Imp)
```

Comment:
seems debug leftover. should remove code comment on line 146

Reply:
removed & thx.